misc/formatter/modules/formatter-core/src/test/java/org/ballerinalang/formatter/core/ParserTestFormatter.java

```diff
@@ -43,4 +104,14 @@ public void test(String fileName, String path) throws IOException {
     public String getTestResourceDir() {
         return Paths.get("parser-tests").toString();
     }
+
+    private String getFilePath(String fileName, String directoryPath) {
```

Comment:
try to return an optional, instead of null

Reply:
Done